middlewares/logger_test.go

```diff
@@ -10,6 +10,8 @@ import (
 	"runtime"
 	"testing"
 
+	"context"
```

Comment:
Please move this import right below the other stdlib ones and use the blank line to separate from the non-stdlib imports following.